pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/jetty/JettyConfig.java

```diff
@@ -1,153 +0,0 @@
-package org.broadinstitute.dsm.jetty;
-
-import javax.servlet.Filter;
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
-
-import org.apache.commons.lang3.StringUtils;
-import org.eclipse.jetty.server.Server;
-import org.eclipse.jetty.server.session.SessionHandler;
-import org.eclipse.jetty.util.thread.QueuedThreadPool;
-import org.eclipse.jetty.util.thread.ThreadPool;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import spark.ExceptionMapper;
-import spark.embeddedserver.EmbeddedServer;
-import spark.embeddedserver.EmbeddedServerFactory;
-import spark.embeddedserver.EmbeddedServers;
-import spark.embeddedserver.jetty.EmbeddedJettyServer;
-import spark.embeddedserver.jetty.HttpRequestWrapper;
-import spark.embeddedserver.jetty.JettyServerFactory;
-import spark.http.matching.MatcherFilter;
-import spark.route.Routes;
-import spark.staticfiles.StaticFilesConfiguration;
-
-public class JettyConfig {
```

Comment:
Moved to [here](https://github.com/broadinstitute/ddp-study-server/pull/2854/files#r1570788809) so to eliminate copy-pasted source.  Now both DSS and DSM can use the same class.